src/platform_impl/windows/event_loop.rs

```diff
@@ -1513,31 +1524,31 @@ unsafe fn public_window_callback_inner<T: 'static>(
                 *SKIP_POINTER_FRAME_MESSAGES,
                 *GET_POINTER_DEVICE_RECTS,
             ) {
-                let pointer_id = LOWORD(wparam as DWORD) as UINT;
-                let mut entries_count = 0 as UINT;
-                let mut pointers_count = 0 as UINT;
-                if GetPointerFrameInfoHistory(
+                let pointer_id = (wparam & 0xffff) as u32; //LOWORD
+                let mut entries_count = 0_u32;
+                let mut pointers_count = 0_u32;
+                if !GetPointerFrameInfoHistory(
                     pointer_id,
-                    &mut entries_count as *mut _,
-                    &mut pointers_count as *mut _,
+                    &mut entries_count,
+                    &mut pointers_count,
                     std::ptr::null_mut(),
-                ) == 0
+                ) == true.into()
```

Comment:
Semantic change (negative comparison to `true` instead of positive comparison to `false`)